dvc/command/pipeline.py

```diff
@@ -167,8 +167,18 @@ def run(self):
 
         pipelines = self.repo.pipelines
         for p in pipelines:
+
+            # Get the stage with the lowest in-degree
+            p_edge = next(filter(lambda x: x[1] == 0, p.in_degree), None)
+            if not p_edge:
+                logger.error("Error finding an edge on the pipeline graph")
```

Comment:
Can this ever happen? If it can, you should raise an error, if it can't then you should remove it and/or put a defensive assert.

Reply:
Fixed thanks @efiop